ProjectManagement/Controllers/BoardsController.cs

```diff
@@ -10,82 +10,78 @@
 
 namespace ProjectManagement.Controllers
 {
-    [Route("api/boards")]
+    [Route("api/[controller]")]
     [ApiController]
-    public class BoardController : ControllerBase
+    public class BoardsController : ControllerBase
     {
         private readonly IBoardService _BoardService;
         private readonly IMapper _mapper;
 
-        public BoardController(IBoardService BoardService, IMapper mapper)
+        public BoardsController(IBoardService BoardService, IMapper mapper)
         {
             _BoardService = BoardService;
             _mapper = mapper;
         }
 
-        [HttpGet("GetBoards")]
+        
+        [HttpGet] // GET api/Boards
         [Authorize]
         public async Task<ActionResult<IEnumerable<BoardDto>>> GetBoardsAsync()
         {
             var items = await _BoardService.GetBoardsAsync();
             var result = _mapper.Map<IEnumerable<BoardDto>>(items);
-
             return Ok(result);
         }
-
-        [HttpGet("GetBoard")]
-        [Authorize]
+ 
+        [HttpGet("{boardId}")] // GET api/Boards/123
+        [Authorize] 
         public async Task<ActionResult<BoardDto>> GetBoardAsync(int boardId)
         {
             Board board = await _BoardService.GetBoardAsync(boardId);
             var result = _mapper.Map<BoardDto>(board);
-
             return Ok(result);
         }
 
-
-        [HttpPost("CreateBoard")]
+        [HttpPost("Create")]// POST api/Boards/Create
```

Comment:
Please, don't use verbs in endpoint names. You don't need to name it "Create", because POST method means creating some resource. So `[HttpPost]` without any parameters should be unough